lib/api/createJob.js

```diff
@@ -0,0 +1,31 @@
+var fs = require("fs");
+var path = require('path');
+var _ = require('underscore');
+var authHelper = require("../util/auth")
+
+/**
+ *  Create jenkins job
+ */
+module.exports = (auth, jobArgs, callback) => {
+    var options = authHelper.createJenkinsOptions(auth.url, auth.user, auth.password);
+    var jenkins = require('jenkins')(options);
```

Comment:
IMHO I would avoid using `require` anywhere but at the top of the file so that you can easily identify what modules this file depends on.